views/js/review/proxy/item.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+/**
+ * Test runner proxy for the QTI item review
+ */
+define([
+    'taoQtiTestPreviewer/previewer/proxy/item',
+], function(qtiItemPreviewerProxy) {
+    'use strict';
+
+    const qtiItemReviewProxy = qtiItemPreviewerProxy;
+    qtiItemReviewProxy.name = 'qtiItemReviewProxy';
```

Comment:
Be careful, you are overwriting the original provider.
It should rather be:
```javascript
const qtiItemReviewProxy = _.defaults({
    name: 'qtiItemReviewProxy',
}, qtiItemPreviewerProxy);
```

Then, on the architecture side, I don't think this is correct. The `itemPreviewer` is meant for a lone item, while the `testPreviewer` should work with a test, so a list of items. Maybe a better approach would have been to add a `test` proxy provider in the `taoQtiTestPreviewer` extension, then use it there.

Reply:
fixed